Lib/test/test_pdb.py

```diff
@@ -2142,6 +2143,118 @@ def test_pdb_asynctask():
             (Pdb) continue
             """
 
+        def test_pdb_await_support():
+            """Testing await support in pdb
+
+            >>> import asyncio
+
+            >>> async def test():
```

Comment:
Can you add tests to check that the awaits correctly awaited using the current contextvars?

Reply:
When a user awaits any coroutine then that coroutine should see the context of the task where pdb.set_trace_async was called, you can create a dummy contextvar and just check its value in and out of the awaited coroutine.

Reply:
I added a test for context var, could you check if that's something you meant?